src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.tsx

```diff
@@ -124,6 +124,7 @@ export const TrackPanelList = () => {
         >
           {selectedTrackPanelTab === TrackSet.VARIATION && (
             <TrackPanelVariantGroupLegend
+              shouldShowLegend={trackCategoryIds.length > 0}
               groups={variantGroups}
               onShowMore={onShowMore}
```

Comment:
I missed this in the previous PR. Why do we still keep the `onShowMore` function in the `TrackPanelList` component? It has a very generic name, but in fact is only concerned with the variant legend. Why not move it to `TrackPanelVariantGroupLegend`?